docs/_posts/2015-09-30-react-render-and-top-level-api.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "React.render and the Top Level React API"
+author: jimandsebastian
+---
+
+
+When you're in React's world you are just building components that fit into other components. Everything is a component. Unfortunately not everything around you is built using React. At the root of your tree you still have to write some plumbing code to connect the outer world into React.
+
+The primary API for rendering into the DOM looks like this:
+
+```js
+ReactDOM.render(reactElement, domContainerNode)
```

Comment:
I like the longer descriptive name here unless we have a rationale. (Maybe we should also rename the argument in the function signature.)